core/src/test/kotlin/api/AdminRouteIntegrationTest.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2022 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.core.api
+
+import io.kotest.assertions.ktor.client.shouldHaveStatus
+
+import io.ktor.client.request.get
+import io.ktor.http.HttpStatusCode
+
+class AdminRouteIntegrationTest : AbstractIntegrationTest({
+
+    "GET /admin/sync-roles" should {
+        "start sync process for permissions and roles" {
+            integrationTestApplication {
+                val response = superuserClient.get("/api/v1/admin/sync-roles")
+                response shouldHaveStatus HttpStatusCode.Accepted
+            }
+        }
+
+        "return Unauthorized if user hve no permissions" {
+            integrationTestApplication {
+                val response = unauthenticatedClient.get("/api/v1/admin/sync-roles")
+                response shouldHaveStatus HttpStatusCode.Unauthorized
+            }
+        }
```

Comment:
Use the helper function for testing permissions instead:
```kotlin
        "require the superuser role" {
            requestShouldRequireRole(Superuser.ROLE_NAME) {
                get("/api/v1/admin/sync-roles")
            }
        }
```

Reply:
Nice. Thanks!